clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/JobController.groovy

```diff
@@ -45,6 +45,7 @@ class JobController {
   @Autowired
   FiatPermissionEvaluator fiatPermissionEvaluator
```

Comment:
You might want to remove this dependency since it's not used any more

Reply:
Thanks @AbdulRahmanAlHamali, great catch!